# Generated by Django 4.2.4 on 2023-08-08 16:40

from django.db import migrations, models


class Migration(migrations.Migration):
    dependencies = [
        ('users', '0004_follow_follow_unique_follow_and_more'),
    ]

    operations = [
        migrations.RemoveConstraint(
            model_name='follow',
            name='cant_subscribe_to_self',
        ),
        migrations.AddConstraint(
            model_name='follow',
            constraint=models.CheckConstraint(
                check=models.Q(
                    ('follower', models.F('following')), _negated=True
                ),
                name='cant_subscribe_to_self',
                violation_error_message='Нельзя подписаться на самого себя',
            ),
        ),
    ]
